manifests/params.pp

```diff
@@ -39,15 +39,23 @@
           $plugin_config_dir           = '/etc/collectd/managed_config'
           $collectd_config_file        = '/etc/collectd/collectd.conf'
           $filtering_config_file       = '/etc/collectd/filtering_config/filtering.conf'
-          $mysql_socket_file           = '/var/lib/mysqld/mysqld.sock'
+          $bundled_plugins             = [
+                                            'collectd-apache',
+                                            'collectd-kafka',
```

Comment:
need to add all other plugins now that this is critical to plugin installation
'collectd-cassandra', 'collectd-java', 'collectd-docker', 'collectd-mongodb', 'collectd-zookeeper',